web/styles/zulip.css

```diff
@@ -2467,21 +2467,17 @@ select.invite-as {
         }
     }
 
-    .column-middle,
     .app-main .column-middle {
         margin-left: 7px;
         margin-right: 7px;
     }
 
-    .header-main .column-middle {
-        margin-left: 0;
-    }
-
-    .column-middle-inner {
+    #navbar-middle {
         margin-left: 0;
     }
 
     .app-main .column-middle .column-middle-inner {
+        margin-left: 0;
```

Comment:
These seem to remove 7px of `margin-right` on the navbar; is that expected?

Reply:
I confirmed that 7x was overridden by the next block which as a more specific selector:
```
        #navbar-middle {                                                                                
            /* = (width of button, square with header) * 4 (number of buttons) + 10px extra margin. */  
            margin-right: calc(var(--header-height) * 4 + 10px);                                        
        }                                                                                               
```